ntoskrnl/ke/amd64/interrupt.c

```diff
@@ -85,14 +85,24 @@ KeConnectInterrupt(IN PKINTERRUPT Interrupt)
     ASSERT(Interrupt->Vector <= MAXIMUM_IDTVECTOR);
     ASSERT(Interrupt->Number < KeNumberProcessors);
     ASSERT(Interrupt->Irql <= HIGH_LEVEL);
+    ASSERT(Interrupt->SynchronizeIrql >= Interrupt->Irql);
+    ASSERT(Interrupt->Irql == (Interrupt->Vector >> 4));
+
+    /* Make sure the vector is within the allowed range */
+    if (Interrupt->Vector < PRIMARY_VECTOR_BASE)
+    {
+        DPRINT1("FIXME: KeConnectInterrupt() with invalid vector 0x%lx!\n", Interrupt->Vector);
+        //__debugbreak();
+        //return FALSE;
```

Comment:
did you actually hit this case ?

Reply:
Yes, it's broken 😄 I'll fix it, once everything is merged. Or maybe The_DarkFire fixes it first.

Reply:
It's no longer broken!